hexane/hybrid_force_matching/md/run.sh

```diff
@@ -1,18 +0,0 @@
-#!/bin/bash -e
-
-MDRUN=mdrun
-GROMPP=grompp
-TRJCONV=trjconv
-
-$GROMPP
-$MDRUN -v
-echo 0 | $TRJCONV -pbc mol -f traj.trr -o fixed_mols.trr -force
-
-csg_fmatch --top topol.tpr --trj fixed_mols.trr --cg hexane.xml --options fmatch_full.xml
```

Comment:
This command is half of the hybrid force matching tutorial, e.g. normal force matching part.